.travis/ubuntu-vanilla.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -xe
+
+# install
+sudo apt update
+sudo apt install -y build-essential cmake3 libgtest-dev
+
+# install Eigen
+#
+# NOTE(vbkaisetsu):
+# Ubuntu 14.04 contains Eigen 3.2
+# primitiv requires 3.3 or later
+#
+wget http://bitbucket.org/eigen/eigen/get/3.3.4.tar.bz2 -O ./eigen.tar.bz2
```

Comment:
Eigen contains only headers, and can be used as below:

```shell
$ EIGEN3_ROOT=/path/to/eigen cmake -DPRIMITIV_...
```

I think some other scripts can be revised similarly.